plugin/mmdb/README.md

```diff
@@ -0,0 +1,56 @@
+# mmdb
+## Name
+*mmdb* - Lookup maxmind geoip2 databases then add associated data to the context request.
+
+## Description
+*mmdb* - Allows you to configure [geoIP2 maxmind databases](https://dev.maxmind.com/geoip/docs/databases) and automatically associate and DNS request data matching the client IP.
+
+Common use cases would be adding per request GEO location data, ISP, ASN, etc.
+
+The data will be added to the request context leveraging the [metadata](../metadata/README.md) plugin, and therefore data will be available by make use of it, label names will be created using the following convention.
+
+## Databases
+The following official [max mind databases](https://dev.maxmind.com/geoip/docs/databases) are allowed.
+
+- Anonymous IP
+- ASN
+- City
+- ConnectionType
+- Country
+- Domain
+- Enterprise
+- ISP
+
+Databases with custom schemas are not allowed, but may be supported in the future.
+
+## Metadatada Label/Value Convention
+**Label**: plugin/mmdb/`KEYNAME`/...
+
+**Value**: Will be the associated field value in its string representation. 
+
+> Note: Numberic types will be represented in base 10.
+
+For example, in a `City` database but not limited to those fields:
+
+| Label                                                 | Value            |
+| :---------------------------------------------------- | :-------------   |
+| `plugin/mmdb/city/names/en`                           | `Cambridge`      |
+| `plugin/mmdb/country/names/en`                        | `United Kingdom` |
+| `plugin/mmdb/continent/names/en`                      | `Europe`         |
+| `plugin/mmdb/registered_country/is_in_european_union` | `false`          |
+| `plugin/mmdb/location/longitude`                      | `0.1315`         |
```

Comment:
can this be sortened? Ie. mmdb/longitude? Or do you get collisions then? What main categories are there? What sub categories are there, i.e. what is the meaning of these label after you've explained the syntax

Reply:
Some fields could be shortened, but others would collide, see example for the City database https://github.com/oschwald/geoip2-golang/blob/main/reader.go#L85-L133 where geoNameId and names among others are used in different nested structs.

My thinking was that is better to provide them all, and let the user decide. Optionally fields could be filtered in/out by users through config.

The full list of fields for all those databases can be found in https://github.com/oschwald/geoip2-golang/blob/main/reader.go#L19-L202

Does it make sense?

Reply:
Simplified using a limited set of fields.